src/pages/shimdongseup/Main/Comment.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+function Comment(props) {
+  return (
+    <div className="comment">
+      <h3 className="comment-user">{props.obj['userId']}</h3>
+      <h4 className="comment-text">{props.obj['comment']}</h4>
+      <div
+        onClick={props.deleteComment}
+        className="material-symbols-outlined delete"
+      >
+        delete
+      </div>
+      <div
+        onClick={props.heartClick}
+        className="material-symbols-outlined heart"
+      >
+        favorite
+      </div>
+    </div>
+  );
+}
```

Comment:
```suggestion
function Comment({ deleteComment, heartClick, obj }) {
  return (
    <div className="comment">
      <h3 className="comment-user">{obj.userId}</h3>
      <h4 className="comment-text">{obj.comment}</h4>
      <div onClick={deleteComment} className="material-symbols-outlined delete">
        delete
      </div>
      <div onClick={heartClick} className="material-symbols-outlined heart">
        favorite
      </div>
    </div>
  );
}
```
- 이렇게 props 부분에서 구조 분해 할당을 해서 진행할 수 있겠네요